ocaml/xapi-idl/example/example.ml

```diff
@@ -38,9 +38,7 @@ let resources =
 
 let socket_path = ref "/var/xapi/socket"
 
-let comma = Re.Str.regexp_string ","
-
-let csv = Re.Str.split_delim comma
+let csv = Astring.String.cuts ~sep:"," ~empty:false
```

Comment:
This could use `String.split_on_char` since we're splitting only on a single character.

Reply:
Unfortunately `split_on_char` allows for empty strings in the list which is not the case for these two.